code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -242,12 +242,15 @@
 	return parts
 
 //Returns a list of damageable limbs
-/mob/living/carbon/human/proc/get_damageable_limbs()
-	var/list/obj/limb/parts = list()
-	for(var/obj/limb/O in limbs)
-		if(O.brute_dam + O.burn_dam < O.max_damage)
-			parts += O
-	return parts
+/mob/living/carbon/human/proc/get_damageable_limbs(var/inclusion_chance)
+    var/list/obj/limb/parts = list()
+    for(var/obj/limb/O in limbs)
+        if(O.brute_dam + O.burn_dam >= O.max_damage)
+            continue
+        if(inclusion_chance && !prob(inclusion_chance))
+            continue
+        parts += O
```

Comment:
Use a more detailed variable name here instead of `O`. Use 'limb' if you have to.